package.json

```diff
@@ -16,6 +16,7 @@
     "chart.js": "^2.9.4",
     "core-js": "^3.17.3",
     "graphql-tag": "^2.12.5",
+
```

Comment:
Not important, but you should consider removing this empty line. `package.json` is supposed to be mostly managed by `npm`.